dotCMS/src/main/java/com/dotcms/rest/api/v1/folder/FolderResource.java

```diff
@@ -91,7 +91,11 @@ public final Response loadFolderByURI(@Context final HttpServletRequest httpServ
         final InitDataObject initData = this.webResource.init(null, httpServletRequest, httpServletResponse, true, null);
         final User user = initData.getUser();
         try{
-            Folder folder = folderHelper.loadFolderByURI(siteName,user,uri);
+
+            String uriParam = uri;
```

Comment:
Maybe this could be something like:
```
final String uriParam = uri.startsWith(StringPool.FORWARD_SLASH) ? StringPool.FORWARD_SLASH.concat(uri) : uri;
final Folder folder = folderHelper.loadFolderByURI(siteName,user,uriParam);
```

Reply:
agree